front/src/applications/operationalStudies/views/Scenario/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -781,6 +785,49 @@ const handleTrainrunOperation = async ({
   }
 };
 
+const handleNoteOperation = async ({
+  type,
+  netzgrafikDto,
+  note,
+  state,
+  dispatch,
+}: {
+  type: NGEEvent['type'];
+  netzgrafikDto: NetzgrafikDto;
+  note: FreeFloatingTextDto;
+  state: MacroEditorState;
+  dispatch: AppDispatch;
+}) => {
+  const noteData = {
+    ngeId: note.id,
+    x: note.x,
+    y: note.y,
+    title: note.title,
+    text: note.text,
+    labelIds: note.labelIds,
+  };
```

Comment:
Do we really need to construct this new object? Couldn't we just pass the unmodified `FreeFloatingTextDto` to `createMacroNote`/`updateMacroNote`/`deleteMacroNote`?